cognite/neat/_utils/cdf/loaders/_data_modeling.py

```diff
@@ -80,8 +80,12 @@ class SpaceLoader(DataModelingLoader[str, SpaceApply, Space, SpaceApplyList, Spa
     resource_name = "spaces"
 
     @classmethod
-    def get_id(cls, item: Space | SpaceApply) -> str:
-        return item.space
+    def get_id(cls, item: Space | SpaceApply | str | dict) -> str:
+        if isinstance(item, Space | SpaceApply):
+            return item.space
+        if isinstance(item, dict):
+            return item["space"]
+        return item
```

Comment:
This change is to make the these methods more flexible, which is useful when parsing error messages.

- [ ]